Program/23BCA281.py

```diff
@@ -0,0 +1,56 @@
+###[ 7 ] Create table contact(fname,laname,contact,email,city). Perform following operation on it
+##1. Insert Contact 8 records
+##2. Delete Contact¶"""
+
+
+import sqlite3 as sq

```

Comment:
What is the role of the sqlite3 module in this program ?

Reply:
I imported a SQLite3 as sq and SQLite3 module is use to interact with the database